README.adoc

```diff
@@ -1,19 +1,25 @@
-= Mobile Docs Previewer
+= Mobile Docs Previewer (docs.aerogear.org)
 
-Allows you preview proposed changes to docs.aerogear.org without installing antora.
+Allows you preview a static html page with proposed changes to docs.aerogear.org without installing Antora.
 
 . Fork this repo.
-. Edit the site.yml to point to your fork and the branch you want to preview, for example:
+. Edit the site.yml to point to your forks of aerogear/mobile-docs and aerogear/mobile-security and specify the branchyou want to preview, for example:
 +
 ----
   - url: https://github.com/finp/mobile-docs.git
     branches: fix-things
+    
+  - url: https://github.com/finp/mobile-security.git
+    branches: fix-things
```

Comment:
is this making it too complex, maybe we should limit the scope of this repo to only mobile-docs, and remove mobile-security from site.yml?

Reply:
keeping with both repos for now

Reply:
well, imo better to keep both

resolved